integration-tests/pom.xml

```diff
@@ -37,10 +37,25 @@
         <maven.javadoc.skip>true</maven.javadoc.skip>
         <maven.install.skip>true</maven.install.skip>
         <maven.deploy.skip>true</maven.deploy.skip>
-        <skip.integration.tests>true</skip.integration.tests>
+        <skip.integration.tests>false</skip.integration.tests>
```

Comment:
Do we want to run these now? I thought the old integration tests are going away soon?

Reply:
Okay, you `@Ignore` all tests now. What's the reason for enabling the tests and then ignoring every test? :smiley:

Reply:
:laughing: The reason was that I investigated, if it would make sense (and how hard it would be) to port the new tests to the new solution. I obviously forgot to switch them back off.